src/App.js

```diff
@@ -50,7 +51,8 @@ export async function decodeFSH(encodedFSH) {
 
 export default function App(props) {
   const classes = useStyles();
-  const text64 = props.match.params;
+  const urlParam = props.match.params;
+  const url = window.location.href;
```

Comment:
Do you know if there's a way to use `props.match` here similar to how we do with the Share functionality? I'm not actually sure if there's a difference, but it seems nice for consistency, and we wouldn't have to use `window`.